Policies/github-acceptable-use-policies.md

```diff
@@ -50,10 +50,21 @@ While using the Service, under no circumstances will you:
 
 - violate the privacy of any third party, such as by posting another person's personal information without consent.
 
-### 4. Services Usage Limits
+### 4. Spam and Inauthentic Activity on GitHub
+Automated excessive bulk activity and coordinated inauthentic activity, such as spamming, are prohibited on GitHub. Prohibited activities include:
+* bulk distribution of promotions and advertising prohibited by GitHub terms and policies
+* inauthentic interactions, such as fake accounts and automated inauthentic activity
```

Comment:
https://github.com/gelstudios/gitfiti 
Will this change block commit history modification and git art?

Reply:
> [gelstudios/gitfiti](https://github.com/gelstudios/gitfiti)
> Will this block commit history modification and git commit art?

It certainly would! That's automated and inauthentic activity.

Reply:
This is actually not a new statement. If you notice, GitHub moved it up from bullet point 9 to bullet point 4, but the same provision exists.

Reply:
it was never a abuse.